source/features/use-first-commit-message-for-new-prs.tsx

```diff
@@ -7,16 +7,14 @@ import features from '.';
 import looseParseInt from '../helpers/loose-parse-int';
 
 async function init(): Promise<void | false> {
-	const commitCount = await elementReady<HTMLElement>([
-		'.overall-summary > ul > li:nth-child(1) .text-emphasized', // Cross fork
-		'[href="#commits_bucket"] .Counter' // Same repository
-	].join());
+	const commitsBucket = await elementReady<HTMLElement>('#commits_bucket');
+	const commitCount = commitsBucket?.previousElementSibling?.querySelector('ul > li:nth-child(1) .text-emphasized');
```

Comment:
😱 

Did you check this on both same-repo and cross-fork?

Reply:
Yes testing link in first post

Reply:
Use `octicon-git-commit`

Reply:
What did you test exactly? Neither one of those commits have a description

Reply:
Your right I did not test the description part but the title changes. Will test it soon.